baseapp/baseapp/activity_log/permissions.py

```
@@ -0,0 +1,25 @@
+
+        if perm == "activity_log.list_profile_activitylog":
+            return True
+
+        if perm == "activity_log.list_node_activitylog":
+            return True
+
+        if perm == "activity_log.view_activitylog":
+            return True
+
+        if perm == "activity_log.view_nodelogevent-data":
+            return True if user_obj.is_superuser else False
```

Ruff

Use `bool(...)` instead of `True if ... else False`

Replace with `bool(...)

(SIM210)